Content.Client/Administration/UI/CustomControls/ConfirmWindow.xaml

```diff
@@ -0,0 +1,8 @@
+<DefaultWindow xmlns="https://spacestation14.io">
+    <BoxContainer Orientation="Vertical" HorizontalAlignment="Center">
+        <GridContainer Columns="2">
```

Comment:
Is there a particular reason you went with a `GridContainer` here, rather than just a `<BoxContainer Orientation="Horizontal">`? That way, you don't have to specify how many child elements there are.